### ISSUE TITLE
Wiki documentation

### ISSUE THREAD

**Issue by nc9** [opennem/opennem#222]
Document OpenNEM on the GitHub wiki in markdown form. A number of pages have already been added:

https://github.com/opennem/opennem/wiki

- [ ] Document getting started with the code
- [ ] Layout of modules
- [ ] Deploying OpenNEM backend
- [ ] Document all networks 
- [ ] Document all data sources
- [ ] Database schema
- [ ] Processing pipelines 
- [ ] Aggregations on backend

**Comment by nc9** [comment]
https://github.com/opennem/opennem/wiki/Network-Data

**Comment by nc9** [comment]
@gmmspi 

I was going to setup an instance of docusaurus on `docs.opennem.org.au`

https://docusaurus.io/docs

merge the wiki and the current dev documentation from `opennem-py/docs` there

**Comment by simonhac** [comment]
i'd prefer we get a bunch of good content together in the wiki before publishing on our website.

**Comment by gmmspi** [comment]
Yes I agree with Simon on that. Documentation update is needed but we can defer this and focus on other things until it's been further developed and reviewed. Might need some public and private docs too - details about the specific hosting services in use may not be publicly interesting / wise to publish.

I'd recommend moving this one to the next version ticket - whatever we tackle it after 3.14.  If that is v4 or 3.15 I'm not sure yet, but we can discuss when we plan figure out the plan for 3.15

**Comment by gmmspi** [comment]
https://ember-climate.org/app/uploads/2022/07/Ember-Electricity-Data-Methodology.pdf

**Comment by nc9** [comment]
https://github.com/opennem/opennem/wiki/Networks

**Comment by nc9** [comment]
https://docs.openelectricity.org.au/
